libraries/matrix/api/src/main/kotlin/io/element/android/libraries/matrix/api/timeline/item/event/EventContent.kt

```diff
@@ -26,7 +27,7 @@ sealed interface EventContent
 
 data class MessageContent(
     val body: String,
-    val inReplyTo: UserId?,
+    val inReplyTo: EventId?,
```

Comment:
Looking at the original data from the Rust SDK and [ReplyToContent](https://github.com/vector-im/element-android/blob/e0b93c2d2cabfbe1f8166f20719df7612670e0cc/matrix-sdk-android/src/main/java/org/matrix/android/sdk/api/session/room/model/relation/ReplyToContent.kt) in EA, I *believe* this should be an `EventId`, but I'm not 100% sure.